.github/workflows/docs-scrapper.yml

```
@@ -3,11 +3,14 @@ name: DocSearch Scrap
 on:
   # Allow to run manually
   workflow_dispatch:
+  # Trigger on successful deployment status
+  deployment_status:
 
 jobs:
   update-docsearch:
     name: Update DocSearch
     runs-on: ubuntu-latest
+    if: github.event_name == 'deployment_status' && github.event.deployment_status.state == 'success' && github.ref == 'refs/heads/master'
```

Do we want to be able to trigger this manually?

```suggestion
    if: github.event_name == 'workflow_dispatch' || (github.event_name == 'deployment_status' && github.event.deployment_status.state == 'success' && github.ref == 'refs/heads/master')
```